src/utils.ts

```diff
@@ -67,6 +70,22 @@ export function optionsFromParameter(parameter: string): Options {
     if (parameter.includes('outputClientImpl=false')) {
       options.outputClientImpl = false;
     }
+
+    if (parameter.includes('nestJs=true')) {
```

Comment:
Notice that the "addGrpcMetadata" and "returnObservable" are ONLY available if its a nestjs run. It really doesn't make a lot of sense for anything but nest.

Reply:
This also keeps things nice and backward compatible.